packages/client/src/pages/RecommendationsPage/RecommendationsTable/RecommendationsTable.tsx

```diff
@@ -62,12 +68,53 @@ const getColumns = (useKilograms: boolean): GridColDef[] => [
   },
 ]
 
+function CustomPagination(
+  handleChange: (event: React.ChangeEvent<{ value: unknown }>) => void,
+) {
+  return () => {
+    const { state, apiRef } = useGridSlotComponentProps()
+    const classes = useStyles()
+
+    return (
+      <GridToolbarContainer>
+        <Box display="flex" flexGrow={1} />
+        <Select
+          value={state.pagination.pageSize}
+          onChange={handleChange}
+          className={classes.selectEmpty}
+        >
+          <MenuItem value={10}>10</MenuItem>
+          <MenuItem value={20}>20</MenuItem>
+          <MenuItem value={50}>50</MenuItem>
+        </Select>
+        <Pagination
+          className={classes.cell}
+          color="primary"
+          size="small"
+          variant="outlined"
```

Comment:
We've decided that the normal Pagination variant looks better. Would you remind removing this line so that it can be the basic variant?